package.json

```diff
@@ -28,6 +28,18 @@
     "/utils/**/*.*(js|d.ts)",
     "!/**/__tests__"
   ],
+  "exports": [
+    "./actions/**/*.*(js|d.ts)",
+    "./auth/**/*.*(js|d.ts)",
+    "./clement/**/*.*(js|d.ts)",
+    "./config/**/*.*(js|d.ts)",
+    "./data/**/*.*(js|d.ts)",
+    "./deprecate/**/*.*(js|d.ts)",
```

Comment:
why do we have `deprecate` here? Seems like we shall not have deprecates in a new lib